code-scanning/anchore.yml

```
@@ -28,6 +28,7 @@ jobs:
     permissions:
       contents: read # for actions/checkout to fetch code
       security-events: write # for github/codeql-action/upload-sarif to upload SARIF results
+      actions: read # only required for a private repository by github/codeql-action/upload-sarif to get the Action run status https://docs.github.com/en/code-security/code-scanning/integrating-with-code-scanning/uploading-a-sarif-file-to-github#example-workflow-for-sarif-files-generated-outside-of-a-repository 
```

```suggestion
      actions: read # only required for a private repository by github/codeql-action/upload-sarif to get the Action run status 
```

The link seems a bit too long. It seems okay to not put link there. I shared it for your reference